wrapper/src/test/resources/logging-test.properties

```
@@ -18,8 +18,8 @@
 .level=INFO
 handlers=java.util.logging.ConsoleHandler
 java.util.logging.ConsoleHandler.level=ALL
-java.util.logging.ConsoleHandler.formatter=java.util.logging.SimpleFormatter
-java.util.logging.SimpleFormatter.format=[%1$tF %1$tT.%1$tL] [%4$-7s] %2$s%n %5$s %n
+java.util.logging.ConsoleHandler.formatter=software.amazon.logging.ExtendedFormatter
+software.amazon.logging.ExtendedFormatter.format=[%1$tF %1$tT.%1$tL] [%4$-7s] [%7$-10s] %2$s%n %5$s %n
```

```suggestion
software.amazon.logging.ExtendedFormatter.format=[%1$tF %1$tT.%1$tL] [%4$-7s] [%7$-10s] %2$s %5$s %n
```

Can we print the entire log on the same line